bootstrap/kubeadm/api/v1alpha2/conversion.go

```diff
@@ -119,7 +119,6 @@ func Convert_v1alpha3_KubeadmConfigStatus_To_v1alpha2_KubeadmConfigStatus(in *ku
 	return nil
 }
 
-
```

Comment:
this whitespace change is not needed. was it generated?

Reply:
It was removed after I executed gofmt -w .

Reply:
And I believe it was not generated by codegen because generated code starts with "// +build !ignore_autogenerated"